### PR TITLE
Add AWX Stack to Community Catalog

### PR DESCRIPTION
**Pull request by MadOtis**
New Catalog template for Ansible AWX (Open-source …version of Ansible Tower).

Includes conditional stack setup of Postgresql, Memcached, and RabbitMQ

### CODE REVIEW

File: templates/awx/0/docker-compose.yml.tpl [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    image: busybox
+    stdin_open: true
+    volumes:
+    - awx-pgdata1:/var/lib/postgresql/data/pgdata
+    tty: true
+    labels:
+      io.rancher.container.start_once: 'true'
+{{- end }}
+  awx-web:
+    image: ansible/awx_web:latest
```
Comment (rawmind0) [inline templates/awx/0/docker-compose.yml.tpl]: Use generic image tag, implicit or explicit, is a bad practice. Could you please use concrete image version??
```
-    image: ansible/awx_web:latest
+    image: ansible/awx_web:1.0
```

File: templates/awx/0/docker-compose.yml.tpl [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+  rabbitmq:
+    image: rabbitmq:3.7.4-alpine
+    environment:
+      RABBITMQ_ERLANG_COOKIE: awx-erlang-cookie
+      RABBITMQ_DEFAULT_VHOST: ${RABBITMQ_VHOST}
+    stdin_open: true
+    tty: true
+{{- end }}
+  awx-task:
+    image: ansible/awx_task:latest
```
Comment (rawmind0) [inline templates/awx/0/docker-compose.yml.tpl]: Use generic image tag, implicit or explicit, is a bad practice. Could you please use concrete image version??
```
-    image: ansible/awx_task:latest
+    image: ansible/awx_task:1.0
```

**Comment by rawmind0** [comment]
Hi @MadOtis ... Thanks for the PR....

Could you please, take a look at changes requested??

**Comment by MadOtis** [comment]
Done!  Thanks for the feedback!

**Comment by rawmind0** [comment]
Thanks for the changes @MadOtis ....

LGTM...

**Pull request merged**